api/proto/info/info.proto

```diff
@@ -507,6 +509,40 @@ message ZInfoNetworkInstance {
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastRefcountChangeTime = 3; // Last time when refCount changed
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
+
+message ZInfoVolume {
+  string uuid = 1;
+  string displayName = 2;
```

Comment:
This, on the other hand, I presume is taken verbatim from the config that EVE received, correct?

Reply:
Yes; we currently have a name in the config